lib/iris/tests/unit/coords/test_DimCoord.py

```diff
@@ -482,5 +484,82 @@ def test_copy_array(self):
         self.assertEqual(coord.bounds[1, 1], 5)
 
 
+class Test_masked__creation(tests.IrisTest, CoordTestMixin):
```

Comment:
As in `test_AuxCoord.py` (see previous note) ...
I think this could all go in the existing `Test__init__`.